__tests__/tools.test.ts

```diff
@@ -0,0 +1,255 @@
+import { Client } from '@modelcontextprotocol/sdk/client/index.js';
+import { convertMcpToolToLangchainTool, loadMcpTools, ResponseFormat } from '../src/tools.js';
```

Comment:
looks like we're never using the import:

`'ResponseFormat' is defined but never used. Allowed unused vars must match /^_/u`

Reply:
Removed